### PR TITLE
Adding remaining HappyBase table helpers.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: gcloud/bigtable/happybase/table.py [hunk 1]
```diff
@@ -26,10 +26,18 @@
 from gcloud.bigtable.column_family import GCRuleIntersection
 from gcloud.bigtable.column_family import MaxAgeGCRule
 from gcloud.bigtable.column_family import MaxVersionsGCRule
+from gcloud.bigtable.happybase.batch import _get_column_pairs
```
Comment (jonparrott) [inline gcloud/bigtable/happybase/table.py]: *grumble* private members again.

**Comment by jonparrott** [comment]
LGTM

File: gcloud/bigtable/happybase/table.py (hunk 1 above)
Comment (dhermes) [inline gcloud/bigtable/happybase/table.py]: You're advocating for reinstatement of `bigtable._helpers` for any helper that gets used in more than one place?

The other one was / is egregious (using a shared sentinel), though this one is justified.

**Comment by dhermes** [comment]
I'm merging with the LGTM but am still interested and your answer to my `bigtable._helpers` question.

File: gcloud/bigtable/happybase/table.py (hunk 1 above)
Comment (jonparrott) [inline gcloud/bigtable/happybase/table.py]: Eh, it's fine.

Comment (dhermes) [inline gcloud/bigtable/happybase/table.py]: Ha. Thanks.

**Comment by jonparrott** [comment]
In general it's just a question of where you draw the "API boundary" - at the module level or at the package level. If you draw it at the package level, then what we're doing now is fine. If you draw it at the module level, then importing private members from other modules is a no-no, because you're overstepping the boundary. IMO, if a module's private function is useful outside of that module it should either be public or moved into a shared dependency.

/opinion /rant

**Pull request merged**